lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -29,7 +29,7 @@ $path= Get-Attr $params "path" $FALSE;
 $regexp = Get-Attr $params "regexp" $FALSE;
 $state = Get-Attr $params "state" "present";
 $line = Get-Attr $params "line" $FALSE;
-$backrefs = Get-Attr $params "backrefs" "no";
+$backrefs = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "backrefs" -default "no")
```

Comment:
in latest powershell.ps1 in devel you can now use -type "bool" which returns you a boolean, so you can avoid the messy looking ConvertTo-Bool call here.